x/feeabs/module.go

```diff
@@ -43,10 +43,13 @@ func (AppModuleBasic) Name() string {
 	return types.ModuleName
 }
 
+func (AppModuleBasic) RegisterCodec(cdc *codec.LegacyAmino) {
```

Comment:
Do we need this func here? I think sdk never call this func

Reply:
Because I see it in TODO list.

Reply:
Hmmm. weird. I think we just need to implement this func `RegisterLegacyAminoCodec`

Reply:
I think we can just remove this func. It not work in sdk